gizmo.py

```diff
@@ -0,0 +1,31 @@
+# hello function
+def hello(name, country = "Finland"):
+    print("Hello", name + ", how are things in", country + "?")
+# spelling function
+def spell(word):
+    for letter in word:
+        print (letter + '.', end = '')
```

Comment:
This prints one additional `.` at the end of the word, eg: `L.i.v.i.a.v.a.n.v.l.i.e.t.`. You need to get rid of that last dot.